packages/kbn-test/src/functional_tests/lib/run_elasticsearch.ts

```diff
@@ -189,19 +193,20 @@ function getESServerlessOptions(
       (config.get('kbnTestServer.serverArgs') as string[])) ||
     [];
 
-  const projectTypeFromArgs = kbnServerArgs
+  const projectType = kbnServerArgs
     .filter((arg) => arg.startsWith('--serverless'))
     .reduce((acc, arg) => {
       const match = arg.match(/--serverless[=\s](\w+)/);
       return acc + (match ? match[1] : '');
-    }, '');
-  const projectType = projectTypeFromArgs.length
-    ? (projectTypeFromArgs as ServerlessProjectType)
-    : undefined;
+    }, '') as ServerlessProjectType;
+
+  if (!isServerlessProjectType(projectType)) {
+    throw new Error(`Unsupported serverless projectType: ${projectType}`);
+  }
```

Comment:
Ops Team question: currently kibana `--serverless` flag is the source to fetch projectType. Do you have any concerns about it? Having it in both ES and Kibana server args seems redundant, just want to check if you see alternative.

Reply:
No concerns.  I don't see any situation where we'll want to run different project types.